frontend/components/SentimentAnalysis.js

```diff
@@ -0,0 +1,73 @@
+import React, {useState} from "react";
+// import * as PropTypes from "prop-types";
+// import STYLES from "./SentimentAnalysis.module.scss";
+
+const SentimentAnalysis = () => {
+    const [userInput, setUserInput] = useState("");
+    const [submitted, setSubmitted] = useState(false);
+    const [audioData, setAudioData] = useState(null);
+
+    const handleInputChange = (event) => {
+        setUserInput(event.target.value);
+    };
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        fetch(`/api/get_sentiment_analysis?text=${userInput}`)
+            .then(response => response.json())
+            .then(data =>{
+                setAudioData(data);
```

Comment:
I'd recommend create two setters here rather than just one (i.e., `setAudioNote` and `setAudioSound` vel sim.). It's always risky to set some arbitrary object into state, since nothing in this file ensures that that object has the properties you'll rely on later. That is, I see below that we access the `note` and `sound` properties of this response data when constructing the UI, but it's not clear here in line 19 that that's what we'd expected. Another developer in the future might reasonably be unsure of what properties we should expect from the response of this fetch, and would otherwise have to look at the API layer to understand that those are the expected properties. 

At the very lease, we can destructure the data response to make it clear here at the fetch what those expected properties are rather than waiting to figure it out in the return of the component. Something like:
```
...
.then(data => {
    { note, sound } = data
    setAudioNote(note)
    setAudioSound(sound)
  }
)
```

Reply:
Would we have to (or should we) slugify the `userInput`? I saw that the test case in PR #2 uses a `text` where spaces are URL-encoded to `%20`. Would this affect the sentiment analysis we run on a text?

Reply:
The URL-encoding doesn't affect the sentiment analysis that we run on the text. In the case of `good%20morning` and `good morning`, the `request.query_params.get('text')` still produces `good morning`.

Reply:
Ah ok, thanks for the clarification!